ios/AirGoogleMaps/AIRGoogleMap.m

```diff
@@ -12,15 +12,38 @@
 #import "RCTConvert+MapKit.h"
 #import "UIView+React.h"
 
-id cameraPositionAsJSON(GMSCameraPosition *position) {
-  // todo: convert zoom to delta lat/lng
+id regionAsJSON(MKCoordinateRegion region) {
   return @{
-           @"latitude": [NSNumber numberWithDouble:position.target.latitude],
-           @"longitude": [NSNumber numberWithDouble:position.target.longitude],
-           @"zoom": [NSNumber numberWithDouble:position.zoom],
```

Comment:
Although, this was released in an rc.0, so we can make a "breaking" change still.